lib/util/Components.js

```diff
@@ -52,8 +52,10 @@ Components.prototype.add = function(node, confidence) {
  * @returns {Object} Component object, undefined if the component is not found
  */
 Components.prototype.get = function(node) {
```

Comment:
I didn't see a good place to put a test for this specific change, and the logic behind `get` changes, the test I did add will fail.